src/game/gameflow.c

```diff
@@ -1272,9 +1281,15 @@ GameFlow_InterpretSequence(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
         case GFS_EXIT_TO_TITLE:
             return GF_EXIT_TO_TITLE;
 
-        case GFS_EXIT_TO_LEVEL:
-            return GF_START_GAME
-                | ((int32_t)(intptr_t)seq->data & ((1 << 6) - 1));
+        case GFS_EXIT_TO_LEVEL: {
+            int32_t next_level =
+                ((int32_t)(intptr_t)seq->data & ((1 << 6) - 1));
```

Comment:
Unnecessary outer braces

Reply:
Needed for `next_level`.

```
../../src/game/gameflow.c:1285:13: error: a label can only be part of a statement and a declaration is not a statement
```

Reply:
I meant the round braces :D

Reply:
Oh lol so parenthesis. 😆 I mix them up all the time too
() parenthesis.
[] brackets.
{} braces.